muxdb/empty_cache.go

```diff
@@ -0,0 +1,22 @@
+package muxdb
+
+import "github.com/vechain/thor/v2/trie"
+
+type emptyCache struct{}
```

Comment:
How about `dummyCache` and put into `cache.go`?

Reply:
Good idea 👍  Fixed!